Modules/Typescript/02_oop_part_1/code/src/Bow.ts

```diff
@@ -0,0 +1,24 @@
+import { Weapon } from './Weapon';
+
+export class Bow extends Weapon {
+  constructor(
+    baseDamage: number,
+    baseDurability: number,
+    value: number,
+    weight: number
+  ) {
+    super('Bow', baseDamage, baseDurability, value, weight);
+  }
+
+  public polish(): void {
+    const maxD: number = 1;
```

Comment:
A readonly constant would be more suitable for this situation

Reply:
wanted to ask you - why? :? I would say - yes, if we would use it as class variable, but when we want to use specific constant in method, it would be better to use just `const` no? :? Also are you suggesting to move this `const` to class level, define it `readonly` that it would be "reusable"/something like configation thingy? (since if we would look just directly to this assignment I wouldn't see benefit of doing something like this or changing to `readonly` :? if we would look in a bigger image, then I would understand that yes - it could be changed :?)

Reply:
Yes, my suggestion was to move the variable to class level for reusability, since this does seem like a constant, i.e. it shouldn't change. The idea does keep in mind the classes extensibility, for this simple example your solution works as well, it was more of a suggestion than an error :)